app/coffeescript/components/mixins/stored_markers.coffee

```diff
@@ -0,0 +1,33 @@
+'use strict'
+
+define [
+  'jquery'
+  'flight/lib/component'
+], (
+  $
+  defineComponent
+) ->
+
+  storedMarkers = ->
+
+    @defaultAttrs
+      storageKey: 'viewedMapMarkers'
+
+    @load = ->
+      return {} unless Storage?
```

Comment:
:+1: 

What do you think about wrapping it in a function so we can clarify the intent? e.g.

```coffee
localStorageSupported = ->
  Storage?
```